app/views/applications/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<h1>Name: <%= @application.name %></h1>
+<h3>Address: <%= @application.address %></h3>
+<h3>Description: <%= @application.description %></h3>
+<h3>Status: <%= @application.status %></h3>
+
+<h2>Links To Pets On Application</h2>
+
+<% @application.pets.each do |pet| %>
+  <ul>
+    <li><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></li>
+  </ul>
+<% end %>
```

Comment:
You always want your `<ul>` on the outside of your `each` statement. You don't want to duplicate the list, just the list items.